src/software/ai/navigator/path_planner/enlsvg_path_planner.cpp

```diff
@@ -70,8 +70,8 @@ std::optional<Path> EnlsvgPathPlanner::findPath(const Point &start,
     EnlsvgPoint enlsvg_end   = convertPointToEnlsvgPoint(end);
 
     // closest unblocked points to requested start and end
-    auto new_start = findClosestUnblockedEnlsvgPoint(enlsvg_start);
-    auto new_end   = findClosestUnblockedEnlsvgPoint(enlsvg_end);
+    auto new_start = findClosestUnblockedEnlsvgPoint(enlsvg_start, enlsvg_end);
```

Comment:
Have we considered if both `enlsvg_start` and `enlsvg_end` are blocked? Is this possible?

Reply:
We do check for that already. If either of the points are blocked and we cant find a nearby unblocked point to, then we just don't create a path.

https://github.com/UBC-Thunderbots/Software/blob/e27d6a8a7ec175a11deda516f5d8187f05fafcde/src/software/ai/navigator/path_planner/enlsvg_path_planner.cpp#L72-L83

Does that answer your question?